doc/src/modules/evalf.rst

```diff
@@ -1,6 +1,6 @@
 .. _evalf-label:
 
-Numerical evaluation
+Numerical Evaluation
 ====================
```

Comment:
Some of these are not really modules and should probably be listed as subtopics under another heading. This one could go under core for example.